### PR TITLE
Composer.json + phpcs

### PR DESCRIPTION
**Pull request by tlovett1**
Does the following:
* Adds composer.json (no autoloading as that breaks things)
* Adds phpcs with the 10up standard
* Fixes phpcs errors
* Adds a gitignore to exclude things

### CODE REVIEW

File: includes/plugins.php [hunk 1]
```diff
@@ -118,17 +126,17 @@ function plugin_meta( $plugin_meta, $plugin_file, $plugin_data, $status ) {
...
 function plugin_deactivation_warning() {
-	$message = __( 'Warning: This plugin provides additional enterprise-grade protective measures such as REST API security and disabling file editing in the dashboard.\n\nAre you sure you want to deactivate?', 'tenup' );
-?>
+	$message = esc_html__( 'Warning: This plugin provides additional enterprise-grade protective measures such as REST API security and disabling file editing in the dashboard.\n\nAre you sure you want to deactivate?', 'tenup' );
+	?>
 <script type="text/javascript">
 jQuery( document ).ready( function( $ ) {
 	$( '.wp-list-table.plugins tr[data-slug="10up-experience"] .deactivate' ).on( 'click', function( e ) {
-		if ( ! window.confirm( '<?php esc_html_e( $message ); ?>' ) ) {
+		if ( ! window.confirm( '<?php echo esc_js( $message ); ?>' ) ) {
```
Comment (helen) [inline includes/plugins.php]: `esc_js()` is really meant for inline JS (like `onclick` attributes) and in modern practice probably doesn't need to be used at all. Why the change?

Comment (tlovett1) [reply includes/plugins.php]: I thought `esc_js` was right here. I could definitely be wrong. The description for the function is "Escape single quotes, htmlspecialchar ” &, and fix line endings.". Escaping single quotes is definitely what we are looking for. I'm pretty sure `window.confirm` doesn't support HTML. Either way `esc_html_e` is definitely not right.

Comment (helen) [reply includes/plugins.php]: The `esc_js()` function itself does some pretty generic stuff, it's really about the intent of it I guess (and also that it's weirdly filterable). It does make sense for the single quotes thing though, I guess this isn't that far off from an inline action.
